lib/BridgedRoom.js

```diff
@@ -261,7 +334,44 @@ BridgedRoom.prototype._handleSlackMessage = function(message, ghost) {
         };
         return ghost.sendMessage(roomID, message);
     }
-    else if (message.files) {
+    else if (subtype === "message_changed") {
+        var previous_message = substitutions.slackToMatrix(message.previous_message.text);
+        var new_message = substitutions.slackToMatrix(message.message.text);
```

Comment:
Both of these can be const.

**EDIT**: Actually,  I think this whole section can be.